requirements/testing/travis_all.txt

```diff
@@ -1,13 +1,22 @@
 # pip requirements for all the travis builds
 
+--find-links https://extras.wxpython.org/wxPython4/extras/linux/gtk3/ubuntu-14.04
+
 codecov
 coverage
-cycler
-numpy
 pillow
 pyparsing
+
+# GUI toolkits
+
+# pgi is only installable on Linux
+pyqt5
+# wxpython 4.0.2 is broken on OSX, see https://groups.google.com/forum/#!topic/wxpython-dev/lNvdniiXhdY
+wxpython!=4.0.2
```

Comment:
Looks like 4.0.3 is also failing now?

Reply:
let's try excluding it too...

Reply:
Fails to build faster!  Actually I'm not clear why the mac build is trying to use 3.7....